### ISSUE TITLE
Invalid Syntax

### ISSUE THREAD

**Issue by garfish316** [keras-team/keras#14539]
epochs = 20
model = Sequential()
model.add(Conv1D(32, 2, activation='relu', input_shape = x_train[0].shape))
model.add(BatchNormalization())
model.add(Droupout(0.2)

model.add(Conv1D(64, 2, activation='relu'))  <<<Invalid Syntax
model.add(BatchNormalization())
model.add(Droupout(0.5)
          
model.add(Flatten())
model.add(Dense(64, activation='relu'))
model.add(Droupout(0.5)

model.add(Dense(1, activation='sigmoid'))

**Comment by bguetarni** [comment]
At line 5, it's `Dropout` and not `Droupout` and you missed a parenthesis: `model.add(Droupout(0.2)` -> `model.add(Droupout(0.2))`.

You misspelled `Dropout` everywhere else also (lines 5, 9 and 13).
